app/controllers/comments_controller.rb

```diff
@@ -47,7 +47,10 @@ def edit
   # POST /comments
   # POST /comments.json
   def create
-    raise if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+    if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+      skip_authorization
```

Comment:
This is necessary in order to opt out from `after_action :verify_authorized`  when we return early.